frontend/src/components/ZipCodeSelector/ZipCodeSelector.tsx

```diff
@@ -1,86 +0,0 @@
-import { without } from 'lodash'
```

Comment:
What's the best practice to keep this code around in case DMHC / California would like to use this?

Reply:
git history :)